debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -181,6 +181,20 @@ public static DecimalHandlingMode parse(String value, String defaultValue) {
             .withImportance(Importance.MEDIUM)
             .withDescription("");
 
+    /**
+     * A comma-separated list of regular expressions that match fully-qualified names of columns to be excluded from monitoring
+     * and change messages. The exact form of fully qualified names for columns might vary between connector types.
+     * For instance, they could be of the form {@code <databaseName>.<tableName>.<columnName>} or
+     * {@code <schemaName>.<tableName>.<columnName>} or {@code <databaseName>.<schemaName>.<tableName>.<columnName>}.
+     */
+    public static final Field COLUMN_WHITELIST = Field.create("column.whitelist")
+            .withDisplayName("Include Columns")
+            .withType(Type.STRING)
+            .withWidth(Width.LONG)
+            .withImportance(Importance.MEDIUM)
+            .withValidation(RelationalDatabaseConnectorConfig::validateColumnBlacklist)
```

Comment:
This validation should go to `COLUMN_BLACKLIST`, right? Would be nice to add a test asserting that validation.

Reply:
Done, and added a test